Dockerfile

```diff
@@ -1,5 +1,12 @@
 FROM onsdigital/flask-crypto-queue
```

Comment:
I think @warrenbailey has built an sdx docker image that already has the relevant libraries - you might want to build from that instead of using onsdigital/flask-crypto-queue, otherwise you are going to be loading the libraries twice!